docs/setting-up/server/docker.md

```diff
@@ -1,44 +1,50 @@
 # Docker
 
-## Introduction
+[TOC]
 
-PMM Server can run as a container with [Docker](https://docs.docker.com) 1.12.6 or later. Images are available at [https://hub.docker.com/r/percona/pmm-server](https://hub.docker.com/r/percona/pmm-server).
+---
+
+Percona maintain a Docker image for PMM Server at <https://hub.docker.com/r/percona/pmm-server>.
 
 The Docker tags used here are for the latest version of PMM 2 ({{release}}) but you can specify any available tag to use the corresponding version of PMM Server.
 
-Metrics collection consumes disk space. PMM needs approximately 1GB of storage for each monitored database node with data retention set to one week. (By default, data retention is 30 days.) To reduce the size of the VictoriaMetrics database, you can consider disabling table statistics.
+## Before you start
+
+- [Install Docker](https://docs.docker.com/get-docker/) 1.12.6 or higher.
+
+- Check disk space: PMM needs approximately 1GB of storage for each monitored database node with data retention set to one week. (By default, data retention is 30 days.) To reduce the size of the VictoriaMetrics database, you can consider disabling table statistics.
 
-Although the minimum amount of memory is 2 GB for one monitored database node, memory usage does not grow in proportion to the number of nodes. For example, 16GB is adequate for 20 nodes.
+- Check RAM: The minimum amount of memory is 2 GB for one monitored database node. (Memory usage does not grow in proportion to the number of nodes. For example, 16GB is adequate for 20 nodes.)
 
-## Run an image
+## Run PMM Server as a Docker image
 
-1. Pull an image.
+1. Pull the image.
 
-    ```bash
+    ```sh
     # Pull the latest 2.x image
     docker pull percona/pmm-server:2
     ```
 
 2. Create a persistent data container.
 
-    ```bash
+    ```sh
     docker create --volume /srv \
     --name pmm-data percona/pmm-server:2 /bin/true
     ```
 
-    !!! caution
+    !!! alert alert-info "Note"
         PMM Server expects the data volume (specified with `--volume`) to be `/srv`.  Using any other value will result in data loss when upgrading.
 
 3. Run the image to start PMM Server.
 
-    ```bash
+    ```sh
     docker run --detach --restart always \
     --publish 80:80 --publish 443:443 \
     --volumes-from pmm-data --name pmm-server \
     percona/pmm-server:2
     ```
 
-    !!! note
+    !!! alert alert-info "Note"
         You can disable manual updates via the Home Dashboard *PMM Upgrade* panel by adding `-e DISABLE_UPDATES=true` to the `docker run` command.
```

Comment:
Not really a todo, but there are many env vars that can be passed in (DISABLE_TELEMETRY for instance)...this may not be the place to put them all but we should do that at some point so that the complete list of options is available and explained...many of these end up being easter-eggs.

Reply:
It IS a TODO. https://jira.percona.com/browse/PMM-5034 is a ticket for detailing envvars. It will be a separate section in 'Details->Environment variables'.